doc/tutorials/using_industry_scenario_on_nuc.rst

```diff
@@ -0,0 +1,105 @@
+
+#. Refer to :ref:`getting-started-building` to set up the development environment
+   for re-compiling the UEFI executable from ACRN source tree.
+
+#. Enter the ``hypervisor`` directory under the ACRN source tree to
+   reconfigure the ACRN hypervisor for industry scenario.Please refer following
+   patch to enable industry scenario.
+   
+   .. code-block:: bash
+	   
+      diff --git a/hypervisor/arch/x86/Kconfig b/hypervisor/arch/x86/Kconfig
+      index 2430ed67..7f92d0fd 100644
+      --- a/hypervisor/arch/x86/Kconfig
+      +++ b/hypervisor/arch/x86/Kconfig
+      @@ -1,6 +1,6 @@
+       choice
+              prompt "ACRN Scenario"
+      -       default SDC
+      +       default INDUSTRY
+              help
+                Select the scenario name to load corresponding VM configuration.
+
+   
+   ``kbl-nuc-i7`` is used for example; you can specify other board type which is closed to your target system.
+
+#. Go to the root of ACRN source tree to build the ACRN UEFI executable
+   with the customized configurations(``kbl-nuc-i7`` board as a example):
+
+    .. code-block:: bash
+
+       $ cd ..
+       $ make FIRMWARE=uefi BOARD=kbl-nuc-i7
+
+#. Copy the generated ``acrn.efi`` executable to the ESP partition (e.g. ``/dev/sda1``).
+   You may need to mount the ESP partition if it's not mounted.
+
+    .. code-block:: bash
+
+       $ sudo mount /dev/sda1 /boot
+       $ sudo cp build/hypervisor/acrn.efi /boot/EFI/acrn/acrn.efi
+
+#. Reboot the ACRN hypervisor and the Service VM.
+
+Launch Guest VMs with Predefined UUIDs
+**************************************
+
+In industry scenario, VMs launched by the ACRN device model ``acrn-dm``
+must match the following UUIDs. You will have to add the ``-U`` parameter
+to the launch script: ``launch_uos.sh`` ``launch_hard_rt_vm.sh`` ``launch_win.sh``,
+in order to attach to specific VM through ``acrn-dm``.
+
+    * d2795438-25d6-11e8-864e-cb7a18b34643
+    * 495ae2e5-2603-4d64-af76-d4bc5a8ec0e5
+    * 38158821-5208-4005-b72a-8a609e4190d0
+	
+For example, the following code snippet is used to launch the Clear Linux Guest VM:
+
+    .. code-block:: none
+       :emphasize-lines: 9
+
+       acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge \
+        -s 2,pci-gvt -G "$3" \
+        -s 5,virtio-console,@stdio:stdio_port \
+        -s 6,virtio-hyper_dmabuf \
+        -s 3,virtio-blk,clear-27550-kvm.img \
+        -s 4,virtio-net,tap0 \
+        $logger_setting \
+        --mac_seed $mac_seed \
+        -U d2795438-25d6-11e8-864e-cb7a18b34643 \
+        -k /usr/lib/kernel/default-iot-lts2018 \
+        -B "root=/dev/vda3 rw rootwait maxcpus=$2 nohpet console=tty0 console=hvc0 \
+        console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M \
+        consoleblank=0 tsc=reliable i915.avail_planes_per_pipe=$4 \
+        i915.enable_hangcheck=0 i915.nuclear_pageflip=1 i915.enable_guc_loading=0 \
+        i915.enable_guc_submission=0 i915.enable_guc=0" $vm_name
+
```

Comment:
The main difference between SDC and industry scenario is the hard RTVM. I think it is better to take launch_hard_rt_vm.sh as example here.

Reply:
Sure, change the example to the content of ``launch_hard_rt_vm.sh`` script